pkg/cmd/kafka/acl/delete/delete.go

```diff
@@ -162,7 +162,7 @@ func runDelete(instanceID string, opts *aclutil.CrudOptions) error {
 		defer httpRes.Body.Close()
 	}
 
-	err = aclutil.ValidateAPIError(httpRes, opts.Localizer, err, "delete", kafkaInstance.GetName())
+	err = aclcmdutil.ValidateAPIError(httpRes, opts.Localizer, err, "delete", kafkaInstance.GetName())
```

Comment:
I think that `ValidateAPIError` belongs more in an SDK instead of a cmd util, but we can revisit that when the error handling refactor happens.